pd/src/staking.rs

```diff
@@ -0,0 +1,33 @@
+use tendermint::{vote, PublicKey};
+
+const PENUMBRA_BECH32_VALIDATOR_PREFIX: &str = "penumbravalpub";
```

Comment:
non-blocking, but I think cosmos uses `cosmosvaloper` or something, maybe we should match it exactly?